README.md

```diff
@@ -26,10 +26,10 @@ In order to connect to your SQL database, you must place a database connector ja
 into $CIRCUS_TRAIN_HOME/lib . Alternatively you can set the environment variable CIRCUS_TRAIN_CLASSPATH to contain the path to the jar file(s).
 
 ### Password Encryption
-Circus Train allows users to provide encrypted passwords in the config. An encrypted password can be generated by doing
+ allows users to provide encrypted passwords in the config. An encrypted password can be generated by doing
 the following: 
  
-    java -cp $CIRCUS_TRAIN_HOME/lib/jasypt-1.9.2.jar  org.jasypt.intf.cli.JasyptPBEStringEncryptionCLI input="Ch4ll3ng3" password=ct_password algorithm=PBEWithMD5AndDES
+    java -cp $HOME/lib/jasypt-1.9.2.jar  org.jasypt.intf.cli.JasyptPBEStringEncryptionCLI input="Ch4ll3ng3" password=ct_password algorithm=PBEWithMD5AndDES
```

Comment:
This should be something like HOUSEKEEPING_HOME so people don't get it confused with user home.